creator-node/src/utils.js

```diff
@@ -434,6 +435,26 @@ async function writeStreamToFileSystem (stream, expectedStoragePath, createDir =
   })
 }
 
+async function runShellCommand (command, args, logger) {
+  return new Promise((resolve, reject) => {
+    const proc = spawn(command, args)
+    let stdout = ''
+    let stderr = ''
+    proc.stdout.on('data', (data) => (stdout += data.toString()))
+    proc.stderr.on('data', (data) => (stderr += data.toString()))
+
+    proc.on('close', (code) => {
+      if (code === 0) {
+        logger.info(`Successfully executed command ${command} ${args} with output: \n${stdout}`)
+        resolve()
+      } else {
+        logger.info(`Error while executing command ${command} ${args} with stdout: \n${stdout}, \nstderr: \n${stderr}`)
```

Comment:
warn?

Reply:
oh this should be error straight up. good catch